server/dba/migrations/46-addForestCords.js

```diff
@@ -0,0 +1,38 @@
+'use strict';
```

Comment:
Can you please apply proper indentation to this whole file?

Reply:
@briandavidson - applied